src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.js

```diff
@@ -37,6 +37,10 @@ function Panel({
   clearHoveredItem,
 }) {
   const [editing, setEditing] = useState(false);
+
+  {
+    /* nosemgrep typescript.react.best-practice.react-props-in-state.react-props-in-state */
```

Comment:
Semgrep complains about line 44, and while I'm all for fixing stuff, it was not immediately clear to me the best fix for this, so it seemed like it was best to ignore for now (your strategy of tagging somebody with a todo would have been better but I didn't think of that!) so that we wouldn't have a perma-failing semgrep task.

Reply:
Oh! Thanks. I was just unfamiliar with "semgrep" so the comment looked bizarre to me 😁